testenrichers/cdi/src/main/java/org/jboss/arquillian/testenricher/cdi/MethodParameterInjectionPoint.java

```diff
@@ -158,6 +158,10 @@ public int getPosition() {
             return new HashSet<Annotation>(Arrays.asList(method.getParameterAnnotations()[position]));
         }
 
+        public <T extends Annotation> Set<T> getAnnotations(Class<T> annotationType) {
```

Comment:
FTR this is actually a default method on implemented interface. However, the source is set to `1.6` which causes the compilation to fail unless you override it.